src/app/scopes.ts

```diff
@@ -246,6 +260,20 @@ export const PermissionScopes: PermissionScope[] = [
     preview: false,
     admin: false,
   },
+  {
+    name: "ProgramControl.Read.All",
+    description: "Read all programs that user can access",
+    longDescription: "Allows the app to read programs and programControls that the signed-in user has access to in the organization",
+    preview: true,
```

Comment:
In the longDescription, programControls is camel cased. Is that the proper way to refer to this feature or is that an internal construct? I suggest changing this to "program controls". Comment applies below as well.

Reply:
Fixed.